features/step_definitions/town_steps.rb

```diff
@@ -46,3 +46,13 @@
   non_feature_branches = Kappamaki.from_sentence(data)
   expect(non_feature_branch_configuration.split(',')).to match_array non_feature_branches
 end
+
+
+Then(/^my non\-feature branches are still not configured$/) do
+  expect(non_feature_branch_configuration.split(',')).to be_empty
+end
+
+
+Then(/^Git Town is (?:no longer|still not) configured for this repository$/) do
+  expect(run('git config --get-regex git-town || true').out).to eql ''
```

Comment:
How about pushing this out into town helpers

```ruby
Then(/^Git Town is (?:no longer|still not) configured for this repository$/) do
  expect(git_town_configurationt).to be_empty
end

def git_town_configuration
  array_output_of 'git config --get-regex git-town || true'
end
```
Using array output as if returns multiple lines if thing are actually in there. Also keeps all the `|| true` limited to the town_helpers file.
```
# when configured
$ git config --get-regex git-town
git-town.main-branch-name master
git-town.non-feature-branch-names release
```